.github/workflows/python-package.yml

```diff
@@ -29,11 +48,17 @@ jobs:
         pip install flake8 pytest
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         python setup.py install
-        # Install Nextflow
-        mkdir /tmp/nextflow && cd /tmp/nextflow
-        wget -qO- get.nextflow.io | bash
-        echo "/tmp/nextflow" >> $GITHUB_PATH
+    - name: Set up Nextflow
+      run: |
+        sudo bash -c "export JAVA_CMD="$(which java)" && cd /usr/bin && (curl -s https://get.nextflow.io | bash) && chmod 755 nextflow"
+        /usr/bin/nextflow -version
     - name: Set up VCF validator and assembly checker
       run: |
         sudo bash -c "cd /usr/bin && wget https://github.com/EBIvariation/vcf-validator/releases/download/v${{ matrix.vcf-validator-version }}/vcf_validator_linux && chmod 755 vcf_validator_linux"
-
+    - name: Set up BCFtools
+      run: |
+        sudo apt update
+        sudo apt -y install maven samtools bcftools parallel libbz2-dev liblzma-dev
```

Comment:
Are you sure you'll get correct version of the tools ?
`samtools` and `bcftools` seems to be at 1.10 in ubuntu 20.04.

Reply:
bcftools compression and CSI indexing seems to work so I didn't worry much about the exact versions..

Reply:
That's fine. The cluster is running different versions anyway: 1.9 for `bcftools` and 1.12 for `samtools`.